app/views/images/_processing.html.erb

```diff
@@ -15,8 +15,20 @@
         </div>
 
         <div class="text-secondary mx-2"><%= image.file_name %></div>
-        <small class="text-muted flex-grow-1">(Processing)</small>
-        <small class="text-muted">(<%= number_to_human_size(image.size) %>)</small>
+
+        <% if image.retryable? %>
+          <div class="flex-grow-1">
+            <%= link_to retry_path, class: "btn btn-sm btn-warning" do %>
```

Comment:
Trying to follow the flow of this retry path, and want to describe what we believe what might be happening:

- We looked at the `episodes_helper`, and found the `episode_retry_image_path`, which sets this path based on whether it's a new upload, or editing an upload
- It also passes some sort of `params[:_retry] = "1"` (meaning `true`) for this retry param value based on which form is being retried

Reply:
Yep - the whole goal is to generate a form `params` something like:

```
podcast[default_feed_attributes][id]: 123
podcast[default_feed_attributes][itunes_images_attributes][0][_retry]: 1
podcast[default_feed_attributes][itunes_images_attributes][0][id]: 456
```

So when rails parses the params, and you call `podcast.assign_attributes(podcast_params)` ... the `accepts_nested_attributes_for` mechanics in the models traverse through, and call `record._retry = 1` on the itunes image.